sdk/servicebus/azure-servicebus/README.md

```diff
@@ -237,11 +237,11 @@ with ServiceBusClient.from_connection_string(connstr) as client:
 When receiving from a queue, you have multiple actions you can take on the messages you receive.
 
 > **NOTE**: You can only settle `ReceivedMessage` objects which are received in `ReceiveMode.PeekLock` mode (this is the default).
-> `ReceiveMode.ReceiveAndDelete` mode removes the message from the queue on receipt.  `PeekedMessage` messages
+> `ReceiveMode.ReceiveAndDelete` mode removes the message from the queue on receipt.  `ReceivedMessage` messages
```

Comment:
Weren't these now `ServiceBusReceivedMessage`? Or has that PR not been merged yet?

Reply:
I guess I missed the part in my old PR :P, updated